src/exceptions/fmt-exception.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <exception licenseId="fmt-exception" name="fmt exception">
```

Comment:
Please add this attribute: `listVersionAdded="3.23"`

Reply:
@zvr - the listVersionAdded tag is not added automatically when published (but interesting thought). The tag is included automatically if you use the online tool to generate the xml which serves as a nice reminder if one has used that tool to submit and make the xml files :)